DirectX11/HackerDXGI.cpp

```diff
@@ -2062,6 +2164,296 @@ STDMETHODIMP HackerDXGISwapChain::GetLastPresentCount(THIS_
 	return hr;
 }
 
+// -----------------------------------------------------------------------------
+
+// HackerUpscalingDXGISwapChain, to provide post-process upscaling to arbitrary
+// resolutions.  Particularly good for 4K passive 3D.
+
+HackerUpscalingDXGISwapChain::~HackerUpscalingDXGISwapChain()
+{
+	if (mFakeSwapChain)
+		mFakeSwapChain->Release();
+	if (mFakeBackBuffer)
+		mFakeBackBuffer->Release();
+}
```

Comment:
Shouldn't this also call through to the superclass?  Just in case anyone adds something to that class?

Probably not virtual, letting it go to default seems better to me.